src/test/java/guitests/PanelsTest.java

```diff
@@ -82,8 +50,60 @@ public void panelsTest() {
         type("dummy2/dummy2");
         push(KeyCode.ENTER);
         pushKeys(CREATE_RIGHT_PANEL);
-        PlatformEx.waitOnFxThread();
-        // Actually a check. If #dummy2/dummy2_col1 did not exist, this would throw an exception.
-        click("#dummy2/dummy2_col1");
+
+        waitUntilNodeAppears(panel1);
```

Comment:
Is this the same as the original? To me it seems that originally we want to assert that a panel can be created after dragging other panels. But in this wait, we just check if the original panel is still present after creating a new panel.